services/core-api/app/api/projects/information_requirements_table/resources/information_requirements_table_list.py

```diff
@@ -1,21 +1,134 @@
+        for idx, row in enumerate(worksheet_to_parse):
+            ## Parse requirements.description from "Information" cell in spreadsheet and cross reference with DB source of truth to determine validity
+            information_cell = row.get('Information', '')
+            # Split Information cell on spaces to separate out description from appendix prefix
+            information_cell_split = information_cell.split()
+            # Rejoin words with spaces after removing appendix reference
+            sanitized_information_cell = ' '.join(information_cell_split[1:]).strip().lower()
+            information_cell_is_valid = valid_requirement_descriptions.count(
+                sanitized_information_cell) > 0
+
+            required_cell = self.convert_boolean_string(row.get('Required'))
+            methods_cell = self.convert_boolean_string(row.get('Methods'))
+            comments_cell = row.get('Comments')
+            # Add 2 to offset zero-based "idx" and starting_row_number beginning at table header
+            row_number = idx + starting_row_number + 2
+            # If "Information" cell entry is not valid flag that to user(could have a bad template or added custom rows)
+            if information_cell_is_valid is False:
+                import_errors.append(
+                    f'Row {row_number} - "{" ".join(information_cell_split[1:]).strip()}" is not a valid entry in the Information column.'
+                )
+                continue
+            # If "Required" or "Methods" cell entry is true and "Comments" are 'None' add error
+            if (required_cell is True or methods_cell is True) and comments_cell == 'None':
+                import_errors.append(
+                    f'Row {row_number} - "Required" or "Methods" cells is checked off and requires "Comments" to be provided.'
+                )
+                continue
+
+            is_empty_row = required_cell is False and methods_cell is False and comments_cell == 'None'
+            active_requirement = [
+                requirement for requirement in valid_requirements
+                if requirement.description.strip().lower() == sanitized_information_cell
+            ]
+            is_row_top_level_category = True if active_requirement[
+                0].parent_requirement_id is None else False
+            # If the "Information" requirement provided does not match DB, an empty row is provided, or the row contains a top level category do not create a requirement
+            if active_requirement and is_empty_row is False and is_row_top_level_category is False:
+                new_requirement_dict = {
+                    'information': sanitized_information_cell,
+                    'requirement_guid': active_requirement[0].requirement_guid,
+                    'required': required_cell,
+                    'methods': methods_cell,
+                    'comments': comments_cell
```

Comment:
should be 'comment' as in irt_requirements_xref table to keep the value in the database.